nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -231,6 +232,27 @@ def setup_transformer_engine_tp_groups(self):
                     tp_group = parallel_state.get_tensor_model_parallel_group()
                     child.set_tensor_parallel_group(tp_group)
 
+    def setup_transformer_engine_cp_groups(self):
+        """ This should be called after context parallel groups have been initialized
+            and only needs to be called when using Transformer Engine.
+        """
+        cp_stream = torch.cuda.Stream()
+
+        for module in self.get_gpt_module_list():
```

Comment:
couldn't find this method. Is it get_model_module_list?

Reply:
please pull the latest commit. this stale code.

Reply:
Thanks. Did you mean latest NeMo context parallel commit or Megatron-LM/TransformerEngine?

Reply:
latest NeMo context parallel commit

Reply:
Thanks!